templates/detect_display_errors.service

```diff
@@ -2,12 +2,11 @@
 Description=Detects display errors
 
 [Service]
-ExecStart=${PROJECT_PATH}/scripts/run/detect_display_errors.sh
+ExecStart=bash "${PROJECT_PATH}"/scripts/run/detect_display_errors.sh
 PIDFile=/var/run/detect_display_errors.pid
 ExecStop=/bin/kill -15 \${MAINPID}
```

Comment:
Quotes around `${MAINPID}`

Reply:
This is actually not standard, at least in systemd service files as the environment variable expansion is handled by systemd.